docs/git-hooks.md

```diff
@@ -0,0 +1,214 @@
+
+To do so, you need to
+- Create the hook under your `resources/git/hooks` folder.
+- Add the hook name to your active hooks under `toolkit.hooks.active`.
+- Create a new class extending the `GitHookCommands` class and define the
+`run` callback for the hook.
+
+### Create the hook
+
+Add the hook that you want to the `resources/git/hooks`, in this example we will
+use the hook `commit-msg`, so we add the file `resources/git/hooks/commit-msg`.
+
+```shell
+#!/bin/sh
+./vendor/bin/run toolkit:hooks-run `basename "$0"` $1 $2
+```
+
+### Add the hook to the active hooks
+
+In your `runner.yml`, extend the Toolkit configuration to add your custom hook.
+
+Do not forget to copy the existing ones (in case you want to use them),
+otherwise your configuration will override the default provided by Toolkit.
+
+```yaml
+toolkit:
+  hooks:
+    active:
+      - pre-commit
+      - prepare-commit-msg
+      - pre-push
+      - commit-msg
+```
+
+### Create a class extending the GitHookCommands
+
+Is in this class that you will define the callback for your hook.
+
+Add your class under `src/TaskRunner/Commands`.
+
+```php
+<?php
+
+namespace Digit\Qa\TaskRunner\Commands;
+
+use EcEuropa\Toolkit\TaskRunner\Commands\GitHookCommands;
+use Robo\ResultData;
+
+class QaGitHookCommands extends GitHookCommands
+{
+    public function runCommitMsg()
+    {
+      $args = $this->input()->getArguments();
+      $commit_message = trim(file_get_contents($args['arg1']));
+      $this->io()->say("Commit message: $commit_message");
+      return ResultData::EXITCODE_OK;
+    }
+}
+```
+
+### Check if your hooks it active
+
+List the available commands, your custom hook should be available.
+
+```
+> ./vendor/bin/run toolkit:hooks-list
++------------------------------+------------------+-------------+---------------+
+| Hook                         | Active by config | Hook exists | Modified file |
++------------------------------+------------------+-------------+---------------+
+| pre-commit (toolkit)         | Yes              | No          | No            |
+| pre-push (toolkit)           | Yes              | No          | No            |
+| prepare-commit-msg (toolkit) | Yes              | No          | No            |
+| commit-msg (digit-qa)        | Yes              | No          | No            |
++------------------------------+------------------+-------------+---------------+
+```
+
+### Other topics
+- [Setting up a project](/docs/setting-up-project.md)
+- [Configuring a project](/docs/configuring-project.md)
+- [Installing the project](/docs/installing-project.md)
+- [Testing the project](/docs/testing-project.md)
+- [Using Docker environment](/docs/docker-environment.md)
+- [Continuous integration](/docs/continuous-integration.md)
+- [Available tasks](/docs/available-tasks.md)
+- [Building assets](/docs/building-assets.md)
+- Git Hooks
+- [Update Project Documentation](/docs/project-documentation.md)
+- [Changelog](/CHANGELOG.md)
```

Comment:
Run phpdoc to update pages.

Reply:
In the documentation was saying to run the phpdoc against the root of the project, this was making the docs to run in the `vendor` folder.
Changed it to run only on `src` folder